fs/operations/rc.go

```diff
@@ -214,7 +214,7 @@ func rcSingleCommand(ctx context.Context, in rc.Params, name string, noRemote bo
 		if err != nil {
 			return nil, err
 		}
-		_, err = CopyURL(ctx, f, remote, url)
+		_, err = CopyURL(ctx, f, remote, url, false)
```

Comment:
We should add another parameter to here (see "rmdirs" just above) which defaults to false (just call in.GetBool and ignore the error)

It should be called the same name as the flag so `autoFilename` and needs to be documented in the copyurl docs (also in this file).

In an ideal world we'd add a bit more in `rc_test.go` in the `TestRcCopyurl` test.

Reply:
Fixed. But it seems i don't understand difference between `rc_test.go` and `operations_test.go`, so maybe i something did wrong here.

Reply:
The rc_test.go tests the operation via the remote control interface which allows us to check the parameters are correct, etc.